src/functions/manager.c

```diff
@@ -20,19 +20,28 @@
 
 */
 
-/** \file */
+/** \file
+ * Singleton class.
+ * Implements connection management.
+ * Listens for incoming connection requests. When one arrives, it parses
+ * the OPEN on a thread to find capabilities and creates a session object and
+ * delegates the session to session_run in the existing thread.
```

Comment:
Bit of a clunky sentence with no commas


```suggestion
 * Listens for incoming connection requests. When one arrives, it parses
 * the OPEN on a thread to find capabilities, creates a session object, and
 * delegates the session to session_run in the existing thread.
```